apis/apps/v1alpha1/componentdefinition_types.go

```diff
@@ -443,6 +444,16 @@ type ComponentDefinitionSpec struct {
 	// +optional
 	UpdateStrategy *UpdateStrategy `json:"updateStrategy,omitempty"`
 
+	// InstanceSet controls the creation of pods during initial scale up, replacement of pods on nodes, and scaling down.
+	//
+	// - `OrderedReady`: Creates pods in increasing order (pod-0, then pod-1, etc). The controller waits until each pod
+	// is ready before continuing. Pods are removed in reverse order when scaling down.
+	// - `Parallel`: Creates pods in parallel to match the desired scale without waiting. All pods are deleted at once
+	// when scaling down.
+	//
+	// +optional
+	PodManagementPolicy *appsv1.PodManagementPolicyType `json:"podManagementPolicy,omitempty"`
```

Comment:
what's the default value of podManagementPolicy? There're two choices:

- default value is OrderedReady, as statefulset
- default value is determined by updateStrategy, the same as current implementation

Reply:
Cannot find the codes to apply the 'PodManagementPolicy', omit something ?

Reply:
See, found it in convertors